packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -209,7 +225,7 @@ export class MobileView extends LitElement {
     <div id="overlay"></div>
     <div class="app">
       <div class="mvContainer">
-        <model-viewer
+        <model-viewer ...=${spread(this.extraAttributes)}
```

Comment:
need to use spread to add extra attributes per: https://open-wc.org/docs/development/lit-helpers/#spread-directives

Reply:
Thanks for figuring this out!